policies/templates/gcp_restricted_firewall_rules_v1.yaml

```
@@ -0,0 +1,627 @@
+                    type: string
+                    enum: ["INGRESS", "EGRESS", "any"]
+                    default: "any"
+                  rule_type:
+                    description: "Type of the rules to restrict."
+                    type: string
+                    enum: ["allowed", "denied", "any"]
+                    default: "any"
+                  port:
+                    description: |
+                      "Port of the rules to restrict. This parameter is tighly coupled with the
+                      protocol paramter: the rule will be marked as in violation if the parameter
+                      protocol:port pair matches one of its
+                      protocol:port listed.
+                      Valid values are:
+                          * single number
+                          * ranges (e.g 22-25) - Note: only FW rules with a range greater or equal
+                            to this passed range would match
+                          * 'any' - when set to any, the ports of the rule will NOT be a factor to
+                                    determine if the FW rule is restricted
+                          * 'all' (if the rule matches all ports for a given protocol)"
+                    type: string
+                    default: "any"
+                  protocol:
+                    description: |
+                      "Protocol of the rules to restrict. This parameter is tighly coupled with
+                      the port paramter: the rule will be marked as in violation if the parameter
+                      protocol:port pair matches one of its protocol:port listed.
+                      Valid values are:
+                          * single protocol (e.g 'tcp', 'icmp' etc.)
+                          * 'any' - when set to any, the protocol of the rule will NOT be a factor
+                                    to determine if the FW rule is restricted
+                          * 'all' (if the rule matches all protocols)"
+                    type: string
+                    default: "any"
+                  source_ranges:
+                    description: |
+                      "Source ranges of the rules to restrict. A fw rule only matches if ONE of its
+                      source CIDR ranges is an exact match of ANY of the CIDR blocks passed in this
+                      parameter.
+                      Valid values are:
+                          * 'X.X.X.X/XX' - i.e valid CIDR blocks
+                          * 'any' - when set to any, the source ranges of the rule will NOT be a factor
+                                    to determine if the FW rule is restricted
+                          * '*' - all CIDR blocks match: this only checks that a source range exists
+                                  in the restricted FW rule"
+                    type: array
+                    items: string
+                    default: "['any']"
```

@AdrienWalkowiak Not sure if this should be a string or an array?

```suggestion
                    default: ['any']
```

---

indeed - fixed.